### DISCUSSION TITLE
建议增加原直播间功能，有些活动直播间过于臃肿

### DISCUSSION THREAD

**Discussion by Lashandy** [the1812/Bilibili-Evolved#2735] [功能建议 / Ideas]
应该就是在网页后增加“?liteVersion=true”，希望再增加默认启用原直播间，感谢

**Comment by Geeyun-JY3** [comment]
这方法是确定通用的吗，之前有确定是通用的切换方法：#2133

这个其实算重复了，如果确实是通用的方法在上述 discussion 下补充即可。

**Reply by Lashandy** [reply]
对，/blanc，应该是通用的，我去试了几个活动直播间都可以切换回原直播间

**Reply by Geeyun-JY3** [reply]
但你这个参数我这边试了一个不行的啊，发个成功的例子？

**Reply by Lashandy** [reply]
比如https://live.bilibili.com/55，KPL活动直播间
在直播间号前面加上/blanc
访问https://live.bilibili.com/blanc/55?liteVersion=true，就会已经发现已经变回原直播间了

**Reply by Geeyun-JY3** [reply]
那这和 `?liteVersion=true` 这个参数没关系啊，是 `/blanc` 的作用。

**Reply by Lashandy** [reply]
我不太懂，只是在其他扩展里看出来区别的😂
